packages/salesforcedx-vscode-apex-replay-debugger/package.nls.json

```diff
@@ -5,6 +5,8 @@
     "A new configuration for launching Apex Replay Debugger",
   "launch_snippet_name": "Launch Apex Replay Debugger",
   "launch_from_log_file": "SFDX: Launch Apex Replay Debugger with Current File",
+  "launch_from_last_log_file":
+    "SFDX: Launch Apex Replay Debugger with Last Log File",
```

Comment:
@ruthemmanuelle - Could you review, if Jim hasn't pinged you offline already? Thanks!